config.yaml

```diff
@@ -29,7 +29,7 @@
 
   # ------------------------------------------
   # Experiment specific info - REQUIRED
-  EXPERIMENT_SET : [experiment_name] # list of the experiments to be executed
+  EXPERIMENT_SET : ["PUM2_K562_ENCSR661ICQ_2"] # list of the experiments to be executed
 
   # Experiment specific info - REQUIRED
   # ***Note: an experiment refers to the foreground (replicates) and background (smis) samples
```

Comment:
I would say "an experimental data set consists of foreground (CLIP) and background samples, possibly multiple samples in each categories. Specify also if replicates are merged.